mne/io/cnt/_utils.py

```diff
@@ -0,0 +1,95 @@
+# Author: Joan Massich <mailsik@gmail.com>
+#
+# License: BSD (3-clause)
+
+from struct import Struct
+from collections import namedtuple
+from math import modf
+from datetime import datetime
+
+from ...utils import warn
+
+
+def _read_teeg(f, teeg_offset):
+    """
+    Read TEEG structure from an open CNT file.
+
+    # from TEEG structure in http://paulbourke.net/dataformats/eeg/
+    typedef struct {
+       char Teeg;       /* Either 1 or 2                    */
+       long Size;       /* Total length of all the events   */
+       long Offset;     /* Hopefully always 0               */
+    } TEEG;
+    """
+    # we use a more descriptive names based on TEEG doc comments
+    Teeg = namedtuple('Teeg', 'event_type total_length offset')
+    teeg_parser = Struct('<Bll')
+
+    f.seek(teeg_offset)
+    return Teeg._make(teeg_parser.unpack(f.read(teeg_parser.size)))
```

Comment:
Using private `_make` does not look clean / safe (using private method). Why not just `return Teeg(*teeg_parser.unpack(f.read(teeg_parser.size)))`

Reply:
'Cos I did not know how to write it! Thx.